docs/rules/no_unneeded_ternary.md

```diff
@@ -0,0 +1,17 @@
+Disallows ternary operators when simpler alternatives exist.
+
+It's a common mistake to use a conditional expression to select between two
+Boolean values instead of using ! to convert the test to a Boolean.
+
+### Invalid:
+
+```typescript
+const foo = condition ? true : false;
+```
+
+### Valid:
+
+```typescript
+const foo = condition ? x : y;
+const foo = condition ? x : false;
+```
```

Comment:
Could you add more examples? I still don't understand the purpose of this rule just from examples

Reply:
Sure. I think I need to add some comments to make it more clear.